src/Strat-LmtOrdrs.py

```diff
@@ -227,13 +233,13 @@ def update_ui(self):
             , loss=self.loss_px)
         self.label_traded.set(tradedstr)
 
-    def create_ui(self):
 
+    def create_ui(self):
         def create_row(header, label, row):
-            Label(self.tk, text=header).grid(row=row,column=0)
-            Label(self.tk, textvariable=label).grid(row=row,column=1)
+            tkinter.Label(self.tk, text=header).grid(row=row, column=0)
+            tkinter.Label(self.tk, textvariable=label).grid(row=row, column=1)
             row += 1
-            return row
+            return row_index
```

Comment:
This should probably return the row and not row_index. row_index is not used in this function